mobile/components/views/common/components/details/review/ReviewTile.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import { Image, Modal, ScrollView, Text, TouchableOpacity, View } from 'react-native';
+import { reviewTile } from './styles';
+import { Opinion } from '../../../../Attractions/types';
+import EditSubMenuModal from '../edit/EditSubMenu';
+
+interface AttractionTileProps {
+  opinion: Opinion;
+  author: { id: number; name: string; avatar: string | undefined };
+  loggedInUserId: number;
+  onPressExtra: () => void;
+  onEdit: () => void;
+}
+
+const AttractionTile: React.FC<AttractionTileProps> = ({
+  opinion,
+  author,
+  loggedInUserId,
+  onPressExtra,
+  onEdit,
+}) => {
+  const [shortReview, setShortReview] = useState(true);
+  const [showImage, setShowImage] = useState(false);
+  const [selectedImage, setSelectedImage] = useState('');
+  const [editMenuVisible, setEditMenuVisible] = useState<boolean>(false);
+  const userId = loggedInUserId;
```

Comment:
You dont need to store variable from props, specially if you are destructing it already on props

Reply:
done